roles/sft-server/molecule/default/converge.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Converge
+  hosts: all
+  tasks:
+    - name: include molecule variables
+      include_vars: shared_vars.yml
+
+    - debug: var=sft_fqdn
+
+    - name: check INSTANCE_UUID is set
+      assert:
+        that:
+          - molecule_instance_id != ""
+        msg: >
+          [ERR] Please run:
+            export INSTANCE_UUID=$RANDOM
+          then re-run this playbook.
+
+    # Normally we assume DNS to be created externally-to-ansible.
+    # Here, we create a DNS record using ansible only for molecule tests.
+    - name: A records
+      route53:
+        command:   create
+        zone:      "{{ sft_root_domain }}"
+        record:    "{{ sft_fqdn }}"
+        type:      A
+        ttl:       60
+        value:     "{{ hostvars[groups['all'][0]].ansible_default_ipv4.address }}"
+        overwrite: yes
+      delegate_to: localhost
+
+    - name: "Include sft-server"
+      include_role:
+        name: "sft-server"
```

Comment:
Is there a technical reason for preferring *include* over *import* here?

Reply:
I will leave this question for @jschaul

Reply:
I have no preference over one or the other. 

> Is there a technical reason for preferring include over import here?

Do you have a preference? The way this question is formulated, it sounds like you would prefer to use 'import'. If that's the case, why not write "import_role is preferred due to XY"? This comment is ambiguous so I cannot act upon it without asking for further explanations first.

Reply:
> I have no preference over one or the other.

It might as well have been chosen intentionally. I just wanted to learn about your reasoning. Depending on your answer, the following statement either would or wouldn't have made sense.

Okay, happy to rephrase this. 

We know, they (*import*a and *include*) behave differently. IMHO, *import* makes more sense in most cases - causing less side effects since *it* does not happen during  'runtime'. Though, it might not be desired in every use case.